core/src/main/java/org/ode4j/ode/internal/joints/DxJointPlane2D.java

```diff
@@ -147,20 +144,25 @@ void getSureMaxInfo( SureMaxInfo info )
 		//    VoXYZ( info.J1a.v[r0], OP.EQ , 0, 0, 0 );
 		//    VoXYZ( info.J1a.v[r1], OP.EQ , 1, 0, 0 );
 		//    VoXYZ( info.J1a.v[r2], OP.EQ , 0, 1, 0 );
-		int rx = info.J1lp + r0; 
-		info._J[rx] = 0; info._J[rx+1] = 0; info._J[rx+2] = 1;
-		rx = info.J1lp + r1;
-		info._J[rx] = 0; info._J[rx+1] = 0; info._J[rx+2] = 0;
-		rx = info.J1lp + r2;
-		info._J[rx] = 0; info._J[rx+1] = 0; info._J[rx+2] = 0;
-//		info.J1l.set(0, 0, 1,  0,0,0,   0,0,0);
-//		info.J1a.set(0, 0, 0,  1,0,0,   0,1,0 );
-		rx = info.J1ap + r0;
-		info._J[rx] = 0; info._J[rx+1] = 0; info._J[rx+2] = 0;
-		rx = info.J1ap + r1;
-		info._J[rx] = 1; info._J[rx+1] = 0; info._J[rx+2] = 0;
-		rx = info.J1ap + r2;
-		info._J[rx] = 0; info._J[rx+1] = 1; info._J[rx+2] = 0;
+		info.setJ1l(0, 0, 0);

```

Comment:
Would it make sense to have a set-method to set more than one value? This looks a bit cumbersome. But then again, I looked at the latest C++ code and they seem to have reduced this to 3 set operations, so maybe it's not worth creating extra methods. I'm happy either way, just wanted to mention it. What do you think?